tests/e2e/pageobjects/dashboard/Dashboard.ts

```diff
@@ -58,11 +61,37 @@ export class Dashboard {
         await this.workspaces.waitWorkspaceListItemAbcence(workspaceName);
     }
 
+     async stopAndRemoveWorkspaceWithSelectingTab(workspaceName: string) {
+        Logger.debug(`Dashboard.stopAndRemoveWorkspaceByUI "${workspaceName}"`);
+        await this.openDashboardTab()
+        await this.stopWorkspaceByUI(workspaceName);
+        await this.workspaces.deleteWorkspaceByActionsButton(workspaceName);
+        await this.workspaces.waitWorkspaceListItemAbcence(workspaceName);
+    }
+
     async openDashboard() {
         Logger.debug('Dashboard.openDashboard');
+        //sometimes we can have opened files. In this case we get Alert dialog. We should perform it.
         await this.driverHelper.getDriver().navigate().to(TestConstants.TS_SELENIUM_BASE_URL);
+        try {
+            await this.waitPage();
+        } catch (err) {
+            if (err instanceof error.UnexpectedAlertOpenError) {
+                this.driverHelper.getDriver().switchTo().alert().accept();
+            } else {
+                throw err;
+            }
+
+        }
         await this.waitPage();
+    }
 
+    // click on the Browser tab which has been registered in the WorkspaceHandlingTests class
+    async openDashboardTab() {
```

Comment:
What if we make the method name more relevant, like `switchToDashboardTab()`, to make the difference with `openDashboard()` method.